README.md

```diff
@@ -396,6 +399,16 @@ application](deploy/common/pmem-app-cache.yaml) example.
 * A node is only chosen the first time a pod starts. After that it will always restart
   on that node, because that is where the persistent volume was created.
 
+Volume requests embedded in Pod spec are provisioned as ephemeral volumes. The volume request could use below fields as [`volumeAttributes`](https://kubernetes.io/docs/concepts/storage/volumes/#csi):
+
+|key|meaning|optional|default value|
+|---|-------|--------|-------------|
+|size|Size of the requested ephemeral volume|No||
+|nsmode|PMEM namespace mode(fsdax/sector) of requested ephemeral volume|Yes|"fsdax"|
```

Comment:
It's not obvious that `fsdax` and `sector` are the valid values. What about this?

|key|meaning|optional|values|
|---|-------|--------|-------------|
|size|Size of the requested ephemeral volume|No||
|nsmode|PMEM namespace mode of requested ephemeral volume|Yes|`fsdax` (default), `sector`|
|eraseAfter|Clear all data after use and before deleting the volume|Yes|`true` (default), `false`|

Reply:
Even better, mark up the keys as `size`, `nsmode` and `eraseAfter`.

Reply:
Done